core/app/assets/javascripts/refinery/core.js.erb

```diff
@@ -1,11 +1,12 @@
 $(document).ready(function(){
   // focus first field in an admin form.
   $('form input[type=text]:first').focus();
-
-  init_flash_messages();
```

Comment:
Yes and i think preview_controller needs refactoring : 

* Missing `:custom_slug` param : https://github.com/refinery/refinerycms/blob/c28e7afae816328280fb6f511094d8cbf100ebd0/pages/app/controllers/refinery/pages/admin/preview_controller.rb#L40-L46

* Hardcoded layout instead of page.layout with fallback : https://github.com/refinery/refinerycms/blob/c28e7afae816328280fb6f511094d8cbf100ebd0/pages/app/controllers/refinery/pages/admin/preview_controller.rb#L36-L38